extensions/amp-ad/0.1/test/test-amp-ad.js

```diff
@@ -32,9 +32,9 @@ describes.realWin('Ad loader', {amp: true}, env => {
     win = env.win;
     doc = win.document;
     a4aRegistryBackup = Object.create(null);
-    Object.keys(a4aRegistry).forEach(k => {
-      a4aRegistryBackup[k] = a4aRegistry[k];
-      delete a4aRegistry[k];
+    Object.keys(AmpAd.a4aRegistry).forEach(k => {
```

Comment:
As above, consider calling `getA4ARegistry` instead of using a static class property.